pennylane_qiskit/converter.py

```diff
@@ -157,23 +246,80 @@ def load(quantum_circuit: QuantumCircuit, measurements=None):
         function: the resulting PennyLane template
     """
 
-    # pylint:disable=fixme, too-many-branches
-    def _function(params: dict = None, wires: list = None):
+    def _function(*args, params: dict = None, wires: list = None, **kwargs):
         """Returns a PennyLane template created based on the input QuantumCircuit.
         Warnings are created for each of the QuantumCircuit instructions that were
         not incorporated in the PennyLane template.
 
+        If the QuantumCircuit contains unbound parameters, this function must be passed
+        parameter values. These can be passed as positional arguments (provided in the same
+        order that they are stored on the QuantumCircuit used to generate this function),
+        keyword arguments with keywords matching the parameter names, or as a single dictionary
+        with the parameters (not parameter names) as keys.
+
         Args:
-            params (dict): specifies the parameters that need to be bound in the QuantumCircuit
+            *args: positional arguments defining the value of the parameters that need to be assigned
+            in the QuantumCircuit
+
+        Kwargs:
+            **kwargs: keyword arguments matching the names of the parameters beind defined
+            params (dict): a dictionary with Parameter keys, specifying the parameters that need
+                to be bound in the QuantumCircuit
             wires (Sequence[int] or int): The wires the converted template acts on.
                 Note that if the original QuantumCircuit acted on :math:`N` qubits,
                 then this must be a list of length :math:`N`.
 
         Returns:
             function: the new PennyLane template
+
+        .. warning::
```

Comment:
I wonder if its worth moving these warnings and examples into the docstring for `load`, as that is the docstring that will render in the website.

Reply:
I think we definitely want them here, because this is what you get if you "ask" the function what it does in Jupyter, i.e.

```
qfunc = qml.from_qiskit(qc)
qfunc?
```

I think we want something more extensive in `qml.from_qiskit`, which will be the documentation on the PennyLane website. @trbromley , do we also want to expand documentation on the `load` function in the plugin, or is that more internal?

Reply:
I like that you can query `qfunc` in Jupyter and get that info! And yes we do want the `from_qiskit` docs to be on point - though we can treat it as a follow up to revisit those docs. I think the `load` function in the plugin can be made a copy of whatever is in `from_qiskit`.